apps/site/src/pages/settings/billing-settings-panel/payment-history-section.tsx

```diff
@@ -0,0 +1,270 @@
+import {
+  Box,
+  styled,
+  Table,
+  TableBody,
+  TableCell as MuiTableCell,
+  TableHead,
+  TableRow,
+  Typography,
+} from "@mui/material";
+import {
+  FunctionComponent,
+  ReactNode,
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from "react";
+import Stripe from "stripe";
+
+import { Link } from "../../../components/link";
+import { LinkButton } from "../../../components/link-button";
+import {
+  getDateFromStripeDate,
+  internalApi,
+} from "../../../lib/internal-api-client";
+import { priceToHumanReadable } from "../../shared/subscription-utils";
+
+const formatInvoiceDate = (date: Date) => {
```

Comment:
Is there a reason not to use `dateToHumanReadable` with the `-` delimitator here?

Reply:
Yes, because the order of the date items is reversed. But have unified the methods to make this re-usable in https://github.com/blockprotocol/blockprotocol/pull/1045/commits/5fe7a074ec1fce723bdc6cbb8a26faa089f32c89